src/CombatantHorizontal.js

```diff
@@ -36,6 +36,8 @@ export default class CombatantHorizontal extends Component {
     // don't need to render this component if this is a limit break
     if (!data.Job && name === 'limit break') return null
 
+    if (!isSelf && config.enableSoloMode) return null
```

Comment:
Please add a comment here like the above

Reply:
Sorry, meant to ask if the comment I added works but accidentally resolved